xml_converter/generators/presubmit.sh

```diff
@@ -1,4 +1,4 @@
-source ./venv/bin/activate
```

Comment:
The virtual environment should be in `xml_converter/generators` not `xml_converter/generators/generators`. Not sure why this was changed.